bukkit/src/main/java/org/popcraft/bolt/listeners/EntityListener.java

```diff
@@ -629,6 +631,13 @@ public void onEntityChangeBlock(final EntityChangeBlockEvent e) {
         if (broken) {
             plugin.removeProtection(protection);
         }
+        // This event is called for waxing or axing a copper block. We need to update the protection to avoid mismatches.
+        // TODO: this might need to be monitor?
```

Comment:
Probably a good idea to make it monitor & it already needs to look it up again anyway for an exact match

maybe name the event handler `onEntityChangeBlockMonitor` or something